pkg/engine/internal/proto/wirepb/wirepb.proto

```diff
@@ -74,10 +75,18 @@ message WorkerHelloMessage {
   uint64 threads = 1;
 }
 
+// WorkerSubscribeMessage is sent by a scheduler to request a WorkerReadyMessage
+// from workers once they have at least one worker thread available.
+//
+// The subscription is cleared once the next WorkerReadyMessage is sent.
+message WorkerSubscribeMessage {}
```

Comment:
do we want to capture the flow of messages in their name? for example, `WorkerReady` and `WorkerHello` are _from_ workers, but `WorkerSubscribe` is now _to_ the workers. Do we want the name to capture this is from the `Scheduler` asking the working for a subscription, and not from the `Worker` asking to subscribe? I'm thinking something like `SchedulerReadyForSubscription` or `SchedulerPing`?